client/src/api/projects.js

```diff
@@ -0,0 +1,19 @@
+import apiRequest from './lib/apiRequest'
+
+const projectsApiClient = {
+  all (apiOptions) {
+    return apiRequest.get('/projects', apiOptions)
+  },
+
+  applied (apiOptions, projectsAppliedFor) {
+    const query = { projectsAppliedFor }
+    return apiRequest.get('/projects', apiOptions, query)
+  },
+
+  organization (apiOptions, id) {
```

Comment:
Hmm.. I think we should try to be more descriptive for these methods. i.e. `getAllProjects`, `getAppliedProjectsForUser`, `getProjectsForOrg`.